components/style/properties/properties.mako.rs

```diff
@@ -2130,10 +2130,6 @@ pub fn modify_style_for_input_text(style: &mut Arc<ServoComputedValues>) {
     margin_style.margin_right = computed::LengthOrPercentageOrAuto::Length(Au(0));
     margin_style.margin_bottom = computed::LengthOrPercentageOrAuto::Length(Au(0));
     margin_style.margin_left = computed::LengthOrPercentageOrAuto::Length(Au(0));
-
-    // whitespace inside text input should not be collapsed
-    let inherited_text = Arc::make_mut(&mut style.inheritedtext);
-    inherited_text.white_space = longhands::white_space::computed_value::T::pre;
```

Comment:
I want to change textarea's `white-space` to `pre-wrap`. However it is set to `pre` for input and textarea here and override any values set in servo.css. I think setting `white-space` in servo.css properly is enough.